### ISSUE TITLE
Please start the backend first.

### ISSUE THREAD

**Issue by Ynantao** [joonspk-research/generative_agents#144]

**Comment by echo-yiyiyi** [comment]
Have you solved it? I also met this issue.

**Comment by Ynantao** [comment]
> Have you solved it? I also met this issue.你解决了吗？我也遇到了这个问题。

就是启动一下后端就可以了

**Comment by echo-yiyiyi** [comment]
> > Have you solved it? I also met this issue.你解决了吗？我也遇到了这个问题。
> 
> 就是启动一下后端就可以了

我启动后端，一旦刷新，还会出现这些问题，你可以跑起来这个项目吗？

**Comment by austinmw** [comment]
Hi, did you solve this?

**Comment by echo-yiyiyi** [comment]
> Hi, did you solve this?

No. Maybe this repo is helpful.https://github.com/drudilorenzo/generative_agents

**Comment by ovjust** [comment]
I also met this issue.
